vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -33,11 +33,19 @@ For more information, see the [Dash documentation](https://dash.plotly.com/exter
 
 ## Add a logo image
 
-Vizro automatically incorporates the dashboard logo in the top-left corner of each page if an image named
-`logo.<extension>` is present within the assets folder.
+### Single logo
 
-Image types of `apng`, `avif`, `gif`, `jpeg`, `jpg`, `png`, `svg`, and `webp` are supported.
+Vizro automatically incorporates a logo in the top-left corner if an image named `logo.<extension>` is present in the
+assets folder.
+
+### Theme-specific logos
+
+You can also supply two images named `logo_dark.<extension>` and `logo_light.<extension>` to switch logos
+based on the theme (dark or light).
+
+Note that both `logo_dark` and `logo_light` must be provided together, and they cannot be combined with a single `logo`.
```

Comment:
```suggestion
Note that both `logo_dark` and `logo_light` must be provided together, and they cannot be combined within a single `logo`.
```

Reply:
Would it make sense to show a screen grab to illustrate what the logo looks like?

Reply:
I like that idea! 👍 

Above is technically incorrect as well, but that's my fault, because I don't think it's clear from my initial draft 😅 
I don't mean that logo_dark and logo_light cannot be combined within a single logo, actually only one will always be visible (depending on which theme is active), so they will be "merged" into one logo.

But the user has the following options:

**Valid Configurations**

- Single Logo: Only logo.<extension> -> will be used for dark and light theme
- Theme Logos: Both logo_dark.<extension> and logo_light.<extension> -> will be switched out for dark / light theme
- No Logo: No logo images provided


And what I meant with the sentence above is that logo_dark.svg and logo_light.svg cannot be provided with logo.svg. How would you phrase that? 😬

Reply:
I've made a suggestion!

Reply:
That's so much better 😄 Thanks for making my explanations clear! 🚀